docs/en/observability/quickstarts/collect-data-with-aws-firehose.asciidoc

```diff
@@ -0,0 +1,64 @@
+[[collect-data-with-aws-firehose]]
+= Quickstart: Collect data with AWS Firehose
+
+preview::[]
+
+In this quickstart guide, you'll learn how to send logs and metrics to Elastic using AWS Firehose.
+
+The AWS Firehose streams are created using the CloudFormation template that can collect all available CloudWatch logs and metrics for your AWS account.
+
+This approach requires minimal configuration as the CloudFormation template creates a Firehose stream, enables CloudWatch metrics collection from all namespaces, and sets up account level subscription filter for CloudWatch log groups to send logs to Elastic via Firehose 
+Optionally, you can run AWS CLI command or upload the template to AWS CloudFormation portal on Amazon to change the values for the parameters.
+
+IMPORTANT: Some AWS services need additional manual configuration to properly ingest logs and metrics. For more information, check the 
+link:https://www.elastic.co/docs/current/integrations/aws[AWS integration] documentation.
+
+Data collection with AWS Firehose is supported on ESS deployments in AWS, Azure and GCP. 
+
+[discrete]
+== Prerequisites
+
+* A deployment using our hosted {ess} on {ess-trial}[{ecloud}]. The deployment includes an {es} cluster for storing and searching your data, and {kib} for visualizing and managing your data.
+* A user with the `superuser` {ref}/built-in-roles.html[built-in role] or the privileges required to onboard data.
++ 
+[%collapsible]
+.Expand to view required privileges
+====
+* {ref}/security-privileges.html#privileges-list-cluster[**Cluster**]: `['monitor', 'manage_own_api_key']`
+* {ref}/security-privileges.html#privileges-list-indices[**Index**]: `{ names: ['logs-*-*', 'metrics-*-*'], privileges: ['auto_configure', 'create_doc'] }`
+* {kibana-ref}/kibana-privileges.html[**Kibana**]: `{ spaces: ['*'], feature: { fleet: ['all'], fleetv2: ['all'] } }`
+====
+* An active AWS account and the necessary permissions to create delivery streams.
+
+[discrete]
+== Limitations
+
+The AWS Firehose receiver has the following limitations:
+
+* It does not support AWS PrivateLink.
+* It is not available for on-premise Elastic Stack deployments. 
+
+[discrete]
+== Collect your data
+
+. In {kib}, go to **Observability** and click **Add Data**.
+
+. Select **Cloud**, **AWS**, and then select **AWS Firehose**.
++
+[role="screenshot"]
+image::images/quickstart-aws-firehose-entry-point.png[AWS Firehose entry point]
+
+. Click **Create Firehose Stream in AWS** to create a CloudFormation stack from the CloudFormation template.
+
+. Go back to the **Add Observability Data** page.
+
+[discrete]
+== Visualize your data
+
+After installation is complete and all relevant data is flowing into Elastic,
+the **Visualize your data** section allows you to access the different dashboards for the various services.
+
+[role="screenshot"]
+image::images/quickstart-aws-firehose-dashboards.png[AWS Firehose dashboards]
```

Comment:
We kind of just drop the user into the abyss here. Instead of adding a screenshot of the various dashboards, would it be better to describe them and why they might be useful?

Reply:
Maybe we don't have to describe all of them. But I think it'd be helpful to provide an example or two of what dashboards are available and why a user might want to look at them.

Reply:
Yeah, I still think there is no need to give an exhaustive mapping between each AWS service and a dashboard we provide, but having a single example of could be useful in case it's a new user and they have not used our dashboards before.

Here is an screenshot for the "VPC Flow" logs dashboard
![CleanShot 2024-11-04 at 10 19 23@2x](https://github.com/user-attachments/assets/a8ef63bf-d523-4480-bdfb-2cb2bbe53ddf)

Reply:
Sounds good to me 👍